cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/SystemArguments.java

```diff
@@ -54,6 +54,8 @@
   private static final String RETRY_POLICY_DELAY_BASE_MS = "system." + Constants.Retry.DELAY_BASE_MS;
   private static final String RETRY_POLICY_DELAY_MAX_MS = "system." + Constants.Retry.DELAY_MAX_MS;
   public static final String TRANSACTION_TIMEOUT = "system.data.tx.timeout";
+  public static final String EMIT_PROGRAM_CONTAINER_METRICS_PREFERENCE = "system.metrics.enabled";
```

Comment:
We usually name the constant based on the value. Better call this `METRICS_ENABLED` and the next `METRICS_CONTEXT_TASK_INCLUDED`